src/tools/scripts/update_timestamps_versions_on_commit.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
I will rename this file and remove `_on_commit` suffix from the name as the script does not decide when to run, it only describes what to do.

Reply:
But it needs a commit sha to run. That's what I meant by "on commit" - not that it's triggered by a commit. Maybe change to "_for_commit"?

Reply:
I am not too picky about it. Long file names are often intimidating, that's why I suggested it. That said, I am okay with whatever you feel will be a good name.

Reply:
OK you persuaded me - filename changed!